.gitignore

```diff
@@ -160,4 +160,7 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
\ No newline at end of file
+#.idea/
+
+# MacOS
+*.DS_Store
```

Comment:
I'd argue that this doesn't belong in a project .gitignore, as this is specific to your environment - instead, you should set up a global `core.excludesfile`. See the [relevant GitHub docs](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).

Reply:
I wouldn't say this is specific to my environment since it's already[ here in the project](https://github.com/nlmatics/llmsherpa/blob/63c60c293ba45d8989a5e67d1a2c3deed6aaa845/.DS_Store) and shouldn't be.
*.DS_Store are temporary MacOS files that are created automatically, so anyone contributing to the project could inadvertently add them. I'm just trying to avoid that in the future